apps/api/src/donations/donations.controller.ts

```diff
@@ -254,9 +254,19 @@ export class DonationsController {
     return this.donationsService.update(id, updatePaymentDto)
   }
 
+  @Post('/invalidate-stripe-payment/:id')
```

Comment:
HTTP `Patch` seems  the more appropriate form of request for this use case.
The endpoint could also be changed to `/:id/invalidate`, as what is being invalidated is existing donation record in our DB, rather than stripe payment. Depending on the implementation, this could also be used to invalidate bank donations.

Reply:
I've tried that but it doesn't work.
There is another Patch( /: id) handler and any request to /:id/invalidate is captured by the first one.

Reply:
Try putting the definition of @Patch (:id/invalidate ) handler above @Patch(:id) one. Should work that way.

Reply:
I tried that as well :)